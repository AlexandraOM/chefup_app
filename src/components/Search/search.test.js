import React from "react";
import { create } from "react-test-renderer";

const SearchTest = (props) => {
    return (
        <div>
            <p>Search for a lesson</p>
            <input type='text'>Search here</input>;
        </div>
        )
};

test("Footer matches the snapshot", () => {
    const search = create(<SearchTest />);
    expect(search.toJSON()).toMatchSnapshot();
});
