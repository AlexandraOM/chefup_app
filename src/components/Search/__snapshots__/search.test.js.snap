// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the snapshot 1`] = `
<div>
  <p>
    Search for a lesson
  </p>
  <input
    type="text"
  >
    Search here
  </input>
  ;
</div>
`;
